File: benchmark/strcat/strcat.cs
using System;

static class Bench
{
    static void Main()
    {
        const int n = 20000000;

        int total = 0;
        for (int i = 0; i < n; i++)
        {
            string s = "id-" + (i % 1000) + "-x";
            total += s.Length;
        }
        Console.WriteLine(total);
    }
}

File: benchmark/strcat/strcat.php
<?php

$n = 20000000;

$total = 0;
for ($i = 0; $i < $n; $i += 1) {
    $s = 'id-' . ($i % 1000) . '-x';
    $total += strlen($s);
}
echo $total, "\n";
